newscoop/include/smarty/campsite_plugins/block.image.php

```diff
@@ -60,11 +60,17 @@ function smarty_block_image(array $params, $content, Smarty_Internal_Template $s
     }
 
     $imageService = \Zend_Registry::get('container')->getService('image');
+    $preferencesService = \Zend_Registry::get('container')->getService('preferences');
+    $caption = $imageService->getCaption($articleRendition->getImage(), $article->number, $article->language->number);
+    if ($preferencesService->get('MediaRichTextCaptions', 'N') == 'N') {
+        $caption = MetaDbObject::htmlFilter($caption);
```

Comment:
I've added escaping here, we didn't do this before. The change makes Newscoop more consistent in it's caption handling, but i'm not sure if it might deliver undesired results for certain clients.